landing/introduction.mdx

```diff
@@ -7,52 +7,43 @@ iconType: "light"
 
 ## What is Particle Network?
```

Comment:
This page is okay as-is, but after launch, need to make it sharper with some nice opening GIF or interactive embedded widget. Something to generate immediate understanding about the product